bin/oneoff/data_fix/fix_bad_school_info_in_users

```diff
@@ -51,18 +52,26 @@ school_info_ids = ActiveRecord::Base.connection.execute(%Q{
 
 ActiveRecord::Base.record_timestamps = false
 
-school_info_ids.each do |school_info_id|
-  users = User.where("user_type = 'teacher' AND school_info_id = ?", school_info_id)
-  next unless users.count > 1
+users_file = "#{Dir.tmpdir}/#{File.basename(__FILE__)}.#{Time.now.strftime('%Y%m%d%H%M')}.csv"
+open(users_file, 'w') do |log|
+  log.puts "id,school_info_id"
```

Comment:
not a big deal, especially with only 2 columns, but note the [CSV](http://ruby-doc.org/stdlib-2.0.0/libdoc/csv/rdoc/CSV.html) class is really handy for things like this. We use it in many places. Here's an [example](https://github.com/code-dot-org/code-dot-org/blob/98290fbf84e9b190645e2b09e53bac138a95c694/bin/cron/update_hoc_map#L29-L28).